### PR TITLE
Added variable to make the path of zone files configureable

### PR DESCRIPTION
**Pull request by leona-ya**
I created a new variable `coredns_zone_files_path` which default is `coredns/zones` and is therefore compatible to previous versions of this role. It would be good to make this configureable because in my case i have an directory in which the playbooks are. Per default ansible will search for the zone files in this playbook directory but I would like to have them in the root directory.

### CODE REVIEW

**Comment by leona-ya** [comment]
Is there noone who could review it? (@paulfantom ?)

File: defaults/main.yml [hunk 1]
```diff
@@ -7,3 +7,5 @@ coredns_config_file: ""
 
 # Use a local binary instead of the official release.
 coredns_binary_local_dir: ""
+
+coredns_zone_files_path: "coredns/zones"
```
Comment (SuperQ) [inline defaults/main.yml]: You need to put the end of file newline back to get the tests to pass.

Let's include the whole fileglob here to make this more extensible.

```suggestion
coredns_zone_files_path:
- "coredns/zones/*"
```

File: tasks/configure.yml [hunk 2]
```diff
@@ -16,4 +16,4 @@
     group: root
     mode: 0644
   with_fileglob:
-    - coredns/zones/*
+    - "{{ coredns_zone_files_path }}/*"
```
Comment (SuperQ) [inline tasks/configure.yml]: And with the list in the variable, we can do this:

```suggestion
  with_fileglob: "{{ coredns_zone_files_path }}"
```

**Review by SuperQ — COMMENTED** [review]
A couple of suggestions

File: tasks/configure.yml (hunk 2 above)
Comment (leona-ya) [reply tasks/configure.yml]: Good suggestions!

File: README.md [hunk 3]
```diff
@@ -37,7 +38,7 @@ Use it in a playbook as follows:
 
 ## Zone files
 
-The role will search in `coredns/zones` in the Ansible templates paths for files to deploy. These can be used with the [`file` plugin](https://coredns.io/plugins/file/).
+The role will search in the path defined in `coredns_zone_files_path` in the Ansible templates paths for files to deploy. These can be used with the [`file` plugin](https://coredns.io/plugins/file/).
```
Comment (SuperQ) [inline README.md]:
```suggestion
The role will search in the path defined in `coredns_zone_files_paths` in the Ansible templates paths for files to deploy. These can be used with the [`file` plugin](https://coredns.io/plugins/file/).
```

**Comment by SuperQ** [comment]
One minor typo, otherwise LGTM.

**Review by SuperQ — APPROVED** [review]
LGTM

**Pull request merged**